### ISSUE TITLE
[question] what other services will be supported once the work to add multiple services is done?

### ISSUE THREAD

**Issue by ninja85a** [TeamNewPipe/NewPipe#995]
I know soundcloud is going to be added but what about other services like twitch.tv?

**Comment by speednroll** [comment]
I was thinking about if it would be possible to add ok.ru as a multiservice in newpipe but I found out that it requires registration, it would be possible anyway?

**Comment by speednroll** [comment]
Sometimes I watch movies on YouTube, but many are censored. add ok.ru and vimeo, to have more content options, sometimes there is content in those pages that can not be seen on youtube :grin:

**Comment by theScrabi** [comment]
I thought about adding login services. We might need to discus this.

**Comment by theScrabi** [comment]
Some even requested spodify.

**Comment by wb9688** [comment]
@theScrabi: But Spotify requires proprietary DRM (Widevine).

**Comment by speednroll** [comment]
@theScrabi I could access these services with a particular newpipe account, which everyone uses. something like yalp store that uses an account to access the content of the play store, since this requires registration

**Comment by speednroll** [comment]
I do not think it is possible to add spotify, this requires a permission of the official application and to play the music you need a premium account if or if.

what could be added is bandcamp, sometimes they leave previews of the songs to listen to them before buying the discs to download them or get the vinyl.

![](https://framapic.org/D2ToxPvpCu9o/fjz3FMZ8jh4k.png)

**Comment by Nutomic** [comment]
Any update on this? Would be nice to have the documentation available somewhere, now that Soundcloud is supported. And do you have a plan which services you are going to add?

**Comment by theScrabi** [comment]
I'm working on a documentation here:
https://github.com/TeamNewPipe/documentation

However during writing it I found some pitholes in the Extractor which I am currently trying to get rid of. So It gets more easy to write additional services.

**Comment by opusforlife2** [comment]
Close. There is no plan. Services are added by whoever wants them.
